jenkins.sh

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env bash
```

Comment:
was this file changed at all?

Reply:
Yeah. A lot of lines changed to add .sh to them. Sad diff algorithm. :(

Reply:
Even using git mv, it doesn't make a nice diff.

Reply:
sad!